shell/common/gin_converters/net_converter.cc

```diff
@@ -334,10 +334,6 @@ bool Converter<scoped_refptr<network::ResourceRequestBody>>::FromV8(
                               static_cast<uint64_t>(offset),
                               static_cast<uint64_t>(length),
                               base::Time::FromDoubleT(modification_time));
-    } else if (type == "blob") {
-      std::string uuid;
```

Comment:
1) Also needs deletion of https://github.com/electron/electron/blob/master/docs/api/structures/upload-blob.md and removal from https://github.com/electron/electron/blob/master/docs/api/structures/post-data.md 


2) Do you think this might be a good opportunity to also address blob type in `uploadData` https://github.com/electron/electron/blob/master/shell/common/gin_converters/net_converter.cc#L275

Reply:
@deepak1556 i was hoping to keep changes to a minimum since this roll's already a bit long in the tooth, but yes, i think that would be an appropriate followup.

Reply:
also re: removal from those docs, i think can still be exposed from a page doing a POST of a blob? i'm not sure how the app would access the blob data though, and it can no longer be forwarded on.

Reply:
Hmm you are right, I missed this section https://github.com/electron/electron/blob/master/docs/api/web-contents.md#event-new-window that allows returning blob data and apps can access them via https://github.com/electron/electron/blob/master/docs/api/session.md#sesgetblobdataidentifier as long as we support it but they can't forward them anymore via loadURL option, so maybe we can just remove the `UploadBlob` type from load url options.

- https://github.com/electron/electron/blob/master/docs/api/webview-tag.md#webviewloadurlurl-options
- https://github.com/electron/electron/blob/master/docs/api/browser-window.md#winloadurlurl-options
- https://github.com/electron/electron/blob/master/docs/api/web-contents.md#contentsloadurlurl-options